pkg/controller/storagecluster/cephblockpools.go

```diff
@@ -28,6 +28,7 @@ func (r *ReconcileStorageCluster) newCephBlockPoolInstances(initData *ocsv1.Stor
 					Size:            3,
 					TargetSizeRatio: .49,
 				},
+				EnableRBDStats: true,
```

Comment:
Why hard-coding it?

Reply:
Why not? If it is OCS recommendation to enable it, and it will consistently provide value, we are well within our rights to simply enable it in all cases. Our goal is to minimize configuration points, after all.

Reply:
Yes, it was requested for this to be always enabled.